code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -100,11 +101,10 @@
 	name = "leporazine medipen"
 	desc = "A rapid way to regulate your body's temperature in the event of a hardsuit malfunction at the cost of some shortness of breath."
 	icon_state = "lepopen"
+	starting_reagent = "leporazine"
 
 /obj/item/weapon/reagent_containers/hypospray/medipen/leporazine/New()
 	..()
-	reagents.remove_reagent("inaprovaline", 10)
-	reagents.add_reagent("leporazine", 9)
 	reagents.add_reagent("lexorin", 1)

```

Comment:
You're going to add 10 leporazine and 1 lexorin into a container that only fits 10 units. This won't work like this.